ui-core/src/stories/ComboBox.stories.tsx

```diff
@@ -22,24 +23,44 @@ const suggestions = [
   { id: '12', label: 'Miguel' },
 ] as Suggestion[];
 
-const meta: Meta<typeof ComboBox> = {
-  component: ComboBox,
+const ComboBoxStory = (props: { small?: boolean; disabled?: boolean; readOnly?: boolean }) => {
+  const [value, setValue] = useState<Suggestion>();
+
+  const getSuggestionLabel = (suggestion: Suggestion) => suggestion.label;
+
+  const onSelectSuggestion = (suggestion?: Suggestion) => {
+    setValue(suggestion);
+  };
+
+  const { filteredSuggestions, onChange, resetSuggestions } = useDefaultComboBox(
```

Comment:
Nit: no need for destructuring here, we could spread the object returned by `useDefaultComboBox()` in `<ComboBox>` like butter. That way, the code is slightly shorter and any new prop returned by `useDefaultComboBox()` in the future will be passed through without any change here.